core/controllers/acl_decorators_test.py

```diff
@@ -1343,6 +1343,85 @@ def test_non_author_cannot_resubmit_suggestion(self):
         self.logout()
 
 
+class DecoratorForAcceptingSuggestionTests(test_utils.GenericTestBase):
+    """Tests for get_decorator_for_accepting_suggestion decorator."""
+    owner_username = 'owner'
```

Comment:
Please use all-capitals for class-level constants. If they clash with out values, just give them a different/more specific name.

Reply:
Done

Reply:
Not done? All these constants should be upper-case.

Reply:
Whoops — was looking at an outdated version. LGTM!